templates/shared_services/airlock_notifier/template_schema.json

```diff
@@ -25,14 +25,22 @@
       "type": "string",
       "title": "SMTP Password",
       "description": "SMTP Password",
-      "updateable": false
+      "updateable": false,
+      "sensitive": true
     },
     "smtp_from_email": {
       "$id": "#/properties/smtp_from_email",
       "type": "string",
       "title": "SMTP From Email",
       "description": "SMTP From Email",
       "updateable": false
+    },
+    "tre_url": {
+      "$id": "#/properties/tre_url",
+      "type": "string",
+      "title": "TRE URL",
+      "description": "If your TRE URL is different from ${TRE_ID}.${LOCATION}.cloudapp.azure.com, please enter it here",
```

Comment:
Maybe this message can be improved? In here I can't substitute TRE_ID and LOCATION with the actual values

Reply:
I think it's ok - maybe add an example of the url? Although you sort of have one there already.

Reply:
Should this also include a validation expression?

Reply:
Added one